### PR TITLE
Level0 hello

### PR DESCRIPTION
**Pull request by MaximKalinin**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module.exports.say = function say(name) 
+{
+  var innerName = "world";
```
Comment (lurenzsgp) [inline src/index.js:3]: Use `let` instead

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  if (typeof name !== "undefined") 
+{
```
Comment (lurenzsgp) [inline src/index.js:5]: 🙈 indentation

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    innerName = name;
+  }
+  return ["Hello ", innerName, "!"].join("");
```
Comment (lurenzsgp) [inline src/index.js:8]: 🙈 new line before

**Review by lurenzsgp — CHANGES_REQUESTED** [review]
Please use 4 spaces of indentation.

File: src/index.js [hunk 4]
```diff
@@ -1,9 +1,10 @@
-module.exports.say = function say(name) 
+module.exports.say = function say(name = "world") 
 {
-  var innerName = "world";
+  let innerName = "world";
   if (typeof name !== "undefined")
```
Comment (lurenzsgp) [inline src/index.js:4]: 🙈 empty line before

I forgot it before the logics block we like an empty line to facilitate the code reading

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+module.exports.say = function say(name) 
+{
+  let innerName = "world";
+  if (typeof name !== "undefined") 
+  {
+    innerName = name;
+  }
```
Comment (lurenzsgp) [inline src/index.js:7]: 🙈 Remove bracket for one-line blocks

File: src/index.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+module.exports.say = function say(name) 
+{
+    let innerName = "world";
+
+    if (typeof name !== "undefined") 
+        innerName = name;
+
+    return ["Hello ", innerName, "!"].join("");
```
Comment (Pigotz) [inline src/index.js:8]: This is fine!

I'll approve this PR but I also suggest you to give a read to [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) article :+1:

**Review by Pigotz — APPROVED** [review]
:+1: 

After the reviews, don't merge the PR, just close it

**Review by lurenzsgp — APPROVED** [review]

**Pull request closed without merging**